imports/plugins/core/orders/client/components/orderSummary.js

```diff
@@ -98,7 +98,7 @@ class OrderSummary extends Component {
             <div className="order-summary-form-group">
               <strong data-i18n="order.created">Created</strong>
               <div className="invoice-details">
-                {moment && moment(order.createdAt).fromNow()} | {dateFormat(order.createdAt, "MM/D/YYYY")}
+                {(moment && moment(order.createdAt).fromNow()) || dateFormat(order.createdAt, "MM/D/YYYY")}
```

Comment:
@Akarshit I think its supposed to show both the `fromNow` date and the `actual date` the `|` probably needed to be like `{"|"}`.

Reply:
@mikemurray If that is the case then the issue is basically moment not being present, which is tracked in #3778

Reply:
@Akarshit That seems to be on the checkout page, whereas this is for the orders dashboard.

Reply:
@mikemurray Sorry I meant #3773.
I get your point I will close this PR.

Reply:
@Akarshit no, I think your PR is fine. #3773 is probably related to the list view. I think you only need to add the `{"|"}` because this is for display is not an OR. It's supposed to display both dates.

The `moment && moment()` is only to keep errors out of the console until `moment` has been loaded.